### ISSUE TITLE
[Infrastructure] Add Play Again Button For Local Games

### ISSUE THREAD

**Issue by Kurt-von-Laven** [freeboardgames/FreeBoardGames.org#903]
**Is your feature request related to a problem? Please describe.**
I find it jarring when local games close abruptly as the winning move is made. The other player may not be looking at the screen and assume you simply hit the back button. This also obfuscates who won if the players do not themselves already know. (Some people prefer to learn by experimentation rather than reading the rules.) In Nine Men's Morris, it is possible to win by eliminating all of the other player's moves, which can take some time to recognize. Closing the game abruptly can rob the losing player of the opportunity to exam the board.

**Describe the solution you'd like**
I would like the game to remain open, declare who won, and show a button the user can click to play the same gain again.

**Describe alternatives you've considered**
An alternative would be to keep the game open and declare who won without adding a play again button.

**Comment by vdfdev** [comment]
I agree we can improve the user experience here. The challenge is that we also want users to find about other games and that was the objective of the game over page re-using the game list of the home page.

**Comment by Kurt-von-Laven** [comment]
I had a feeling that was the rationale, and I like that “don’t get addicted to this one game” vibe myself. One option might be to have a dedicated game over view that keeps the game around and shows who won, but also some suggestions for other games to play? Another may be to reuse the game list, but stick the last screen of the game with a message about who won at the top?
